copier/vcs.py

```diff
@@ -190,12 +189,12 @@ def clone(url: str, ref: str | None = None) -> str:
             _clone = _clone["--filter=blob:none"]
     _clone()
     # Include dirty changes if checking out a local HEAD
-    if ref in {None, "HEAD"} and os.path.exists(url) and Path(url).is_dir():
+    url_abspath = Path(url).absolute()
+    if ref in {None, "HEAD"} and url_abspath.exists() and url_abspath.is_dir():
```

Comment:
I believe `is_dir` implies `exists`.

```suggestion
    if ref in {None, "HEAD"} and url_abspath.is_dir():
```

Reply:
Thanks for making sure of it!